docs/reference/aggregations/bucket/frequent-items-aggregation.asciidoc

```diff
@@ -144,19 +146,28 @@ example.
 -------------------------------------------------
 POST /kibana_sample_data_ecommerce/_async_search
 {
-  "size": 0,
-  "aggs": {
-    "my_agg": {
-      "frequent_items": {
-        "minimum_set_size": 3,
-        "fields": [
-          { "field": "category.keyword" }, 
-          { "field": "geoip.city_name" }
-        ],
-        "size": 3
+   "size":0,
+   "aggs":{
+      "my_agg":{
+         "frequent_items":{
+            "minimum_set_size":3,
+            "fields":[
+               {
+                  "field":"category.keyword"
+               },
+               {
+                  "field":"geoip.city_name",
+                  "exclude":"other"
+               }
+            ],
+            "size":3,
+            "terms":{
```

Comment:
this `terms` addition looks wrong to me, a C&P error?

Reply:
Yup, it is! Thank you for spotting it. Addressed via [fc03408](https://github.com/elastic/elasticsearch/pull/92758/commits/fc03408304098de20af51d44fd0593b9dd46cd71).